voice/methods/recordings/getCallsCallIdRecordingsRecordingIdTranscription.md

```diff
@@ -0,0 +1,148 @@
+{% method %}
+
+## Download Transcription
+
+Retrieve the specified recording's transcription file. [⚠️ Be sure to not expose your API Credentials to end-users](./about.md#caution-recordings)
+
+### Request URL
+
+<code class="get">GET</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recordings/{recordingId}/transcription`
+
+#### Basic Authentication
+
+Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Credentials. Read more about how Bandwidth secures endpoints in the [Security & Credentials](../../../guides/accountCredentials.md) document.
+
+---
+
+### Supported Parameters
+
+| Parameter | Description |
+|:----------|:------------|
+| None      | None        |
+
+
+### Transcription Attributes
+
+<!--
+Description taken from:
+https://docs.aws.amazon.com/transcribe/latest/dg/API_streaming_Item.html
+https://docs.aws.amazon.com/transcribe/latest/dg/transcribe-dg.pdf
```

Comment:
Should we put this here? I'm fine with it, but earlier it sounded like we don't want to advertise our transcription provider.

Reply:
oops, I thought about leaving this here for our reference, but it does go as an html comment in the rendered doc. I'll remove it from here.
Thanks for noticing.